mediagalaxy.ro,"HTTPSConnectionPool(host='mediagalaxy.ro', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mediagalaxy.ro', port=80): Read timed out. (read timeout=2)"
medicalmedia.co.il,"HTTPSConnectionPool(host='medicalmedia.co.il', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5a011490>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='medicalmedia.co.il', port=80): Read timed out. (read timeout=2)"
mediclinic.ae,"HTTPSConnectionPool(host='mediclinic.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.mediclinic.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
medinet.gov.vn,"HTTPSConnectionPool(host='medinet.gov.vn', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0f5aac5670>, 'Connection to medinet.gov.vn timed out. (connect timeout=2)'))","HTTPConnectionPool(host='medinet.gov.vn', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0f59c7a970>, 'Connection to medinet.gov.vn timed out. (connect timeout=2)'))"
medvestnik.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
medwave.cl,"HTTPSConnectionPool(host='medwave.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f59038ac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='medwave.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f58f6f8b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mefos.hr,"HTTPSConnectionPool(host='mefos.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f57f2c820>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mefos.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f58589880>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
megafon-news.co.il,"HTTPSConnectionPool(host='megafon-news.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='megafon-news.co.il', port=443): Read timed out. (read timeout=2)"
megashop.hr,"HTTPSConnectionPool(host='www.megashop.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.megashop.hr', port=443): Read timed out. (read timeout=2)"
megashop.si,"HTTPSConnectionPool(host='www.megashop.si', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.megashop.si', port=443): Read timed out. (read timeout=2)"
megatv.com,"HTTPSConnectionPool(host='megatv.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.megatv.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
meiji.ac.jp,"HTTPSConnectionPool(host='meiji.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f56e8d7c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='meiji.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f5760c280>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
melbourne.vic.gov.au,"HTTPSConnectionPool(host='melbourne.vic.gov.au', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0f561d4f40>, 'Connection to melbourne.vic.gov.au timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='www.melbourne.vic.gov.au', port=443): Read timed out. (read timeout=2)"
melody4arab.xyz,"HTTPSConnectionPool(host='melody4arab.xyz', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f55ecf940>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='melody4arab.xyz', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f562b81f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
melonbooks.co.jp,"HTTPSConnectionPool(host='melonbooks.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: EE certificate key too weak (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.melonbooks.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
men.gov.ma,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
mendoza.gov.ar,"HTTPSConnectionPool(host='mendoza.gov.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5813c520>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mendoza.gov.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f56012d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menlhk.go.id,"HTTPSConnectionPool(host='menlhk.go.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f57071d90>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menlhk.go.id', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f5b4854f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menofia.edu.eg,"HTTPSConnectionPool(host='menofia.edu.eg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f56b89e80>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menofia.edu.eg', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f55ecf6d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menoinfo.fi,"HTTPSConnectionPool(host='menoinfo.fi', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5941a0a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menoinfo.fi', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f55ef4520>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menupages.ie,"HTTPSConnectionPool(host='menupages.ie', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0f5b7e6b80>, 'Connection to menupages.ie timed out. (connect timeout=2)'))","HTTPConnectionPool(host='menupages.ie', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0f570c7f40>, 'Connection to menupages.ie timed out. (connect timeout=2)'))"
mep.go.cr,Exceeded 30 redirects.,Exceeded 30 redirects.
mercadopublico.cl,"HTTPSConnectionPool(host='mercadopublico.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5a24c7f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mercadopublico.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f5b7911f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mercedes.gob.ar,"HTTPSConnectionPool(host='mercedes.gob.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5b22e040>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mercedes.gob.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f57362160>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
meteo.it,"HTTPSConnectionPool(host='meteo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='meteo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
meteo.lt,"HTTPSConnectionPool(host='meteo.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='meteo.lt', port=80): Read timed out. (read timeout=2)"
meti.go.jp,"HTTPSConnectionPool(host='meti.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f56db76a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='meti.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f533f4f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
metjekutindevut.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
metro.tokyo.jp,"HTTPSConnectionPool(host='metro.tokyo.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f530aaac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='metro.tokyo.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f52dde040>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
metropolitanmagazine.it,"HTTPSConnectionPool(host='metropolitanmagazine.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='metropolitanmagazine.it', port=443): Read timed out. (read timeout=2)"
metrouniv.ac.id,"HTTPSConnectionPool(host='metrouniv.ac.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='metrouniv.ac.id', port=443): Read timed out. (read timeout=2)"
mext.go.jp,"HTTPSConnectionPool(host='mext.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5928ca60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mext.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f5a240b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mfa.ee,"HTTPSConnectionPool(host='mfa.ee', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f561072b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mfa.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f574e7df0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mfa.gov.sg,"HTTPSConnectionPool(host='mfa.gov.sg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mfa.gov.sg' doesn't match 'www.mfa.gov.sg'"")))","HTTPSConnectionPool(host='mfa.gov.sg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mfa.gov.sg' doesn't match 'www.mfa.gov.sg'"")))"
mfa.gov.tr,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPSConnectionPool(host='mfa.gov.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mfe.govt.nz,"HTTPSConnectionPool(host='environment.govt.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='environment.govt.nz', port=443): Read timed out. (read timeout=2)"
mhlw.go.jp,"HTTPSConnectionPool(host='mhlw.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f54407b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mhlw.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f52cef520>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
micheleabruzzese.it,"HTTPSConnectionPool(host='micheleabruzzese.it', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='micheleabruzzese.it', port=80): Read timed out. (read timeout=2)"
midad.com,"HTTPSConnectionPool(host='midad.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPConnectionPool(host='midad.com', port=80): Read timed out. (read timeout=2)"
mides.gub.uy,"HTTPSConnectionPool(host='mides.gub.uy', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0f5c6ddbe0>, 'Connection to mides.gub.uy timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mides.gub.uy', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0f59c00fd0>, 'Connection to mides.gub.uy timed out. (connect timeout=2)'))"
mie.lg.jp,"HTTPSConnectionPool(host='mie.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5326f850>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mie.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f527a4fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
migdalor-news.co.il,"HTTPSConnectionPool(host='www.migdalor-news.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.migdalor-news.co.il', port=443): Read timed out. (read timeout=2)"
mikmak.co.il,"HTTPSConnectionPool(host='mikmak.co.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0f523362b0>, 'Connection to mikmak.co.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mikmak.co.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0f5994aeb0>, 'Connection to mikmak.co.il timed out. (connect timeout=2)'))"
mikronis.hr,"HTTPSConnectionPool(host='mikronis.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.mikronis.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
milejemplos.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
militaar.net,"HTTPSConnectionPool(host='militaar.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPConnectionPool(host='militaar.net', port=80): Read timed out. (read timeout=2)"
milog.co.il,"HTTPSConnectionPool(host='milog.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='milog.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
min-mave.dk,"HTTPSConnectionPool(host='min-mave.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='min-mave.dk', port=443): Read timed out. (read timeout=2)"
min-saude.pt,"HTTPSConnectionPool(host='min-saude.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5bcf7730>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='min-saude.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f524a63a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minci.gob.ve,"HTTPSConnectionPool(host='minci.gob.ve', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='minci.gob.ve', port=80): Read timed out. (read timeout=2)"
minciencias.gov.co,"HTTPSConnectionPool(host='minciencias.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='minciencias.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mincyt.gob.ar,"HTTPSConnectionPool(host='mincyt.gob.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5abd1df0>: Failed to establish a new connection: [Errno 113] No route to host'))","HTTPSConnectionPool(host='www.argentina.gob.ar', port=443): Read timed out. (read timeout=2)"
mindef.gob.bo,"HTTPSConnectionPool(host='mindef.gob.bo', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mindef.gob.bo', port=80): Read timed out. (read timeout=2)"
mindop.sk,"HTTPSConnectionPool(host='mindop.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='mindop.sk', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0f590387c0>, 'Connection to mindop.sk timed out. (connect timeout=2)'))"
minec.gob.sv,"HTTPSConnectionPool(host='minec.gob.sv', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0f59d8beb0>, 'Connection to minec.gob.sv timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.minec.gob.sv', port=80): Read timed out. (read timeout=2)"
minec.gob.ve,"HTTPSConnectionPool(host='minec.gob.ve', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5b21aca0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minec.gob.ve', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f592c2910>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mineco.gob.gt,"HTTPSConnectionPool(host='mineco.gob.gt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f53024430>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mineco.gob.gt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f52dde280>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mined.gob.sv,"HTTPSConnectionPool(host='mined.gob.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f544a1b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mined.gob.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f574e7cd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minedu.gob.bo,"HTTPSConnectionPool(host='minedu.gob.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='minedu.gob.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
minedu.sk,"HTTPSConnectionPool(host='minedu.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5b21a580>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minedu.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f522e50d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mineducacion.gov.co,"HTTPSConnectionPool(host='mineducacion.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5247cf70>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='mineducacion.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f52295820>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
minetilbud.dk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
minfin.bg,"HTTPSConnectionPool(host='www.minfin.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.minfin.bg', port=443): Read timed out. (read timeout=2)"
minicvaldes.com,"HTTPConnectionPool(host='www.minicvaldes.com', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.minicvaldes.com', port=80): Read timed out. (read timeout=2)"
mininterior.gov.co,"HTTPSConnectionPool(host='mininterior.gov.co', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0f53cab8b0>, 'Connection to mininterior.gov.co timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mininterior.gov.co', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0f58533a30>, 'Connection to mininterior.gov.co timed out. (connect timeout=2)'))"
ministeriodegobierno.gob.ec,"HTTPSConnectionPool(host='ministeriodegobierno.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ministeriodegobierno.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
ministeriodesalud.go.cr,"HTTPSConnectionPool(host='ministeriodesalud.go.cr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f56e02c70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ministeriodesalud.go.cr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f53e01d90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minsa.gob.pe,"HTTPSConnectionPool(host='minsa.gob.pe', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f57324910>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minsa.gob.pe', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f5a6a2040>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mintrabajo.gov.co,"HTTPSConnectionPool(host='mintrabajo.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f56153ac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mintrabajo.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f5ad73c70>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mintransporte.gov.co,"HTTPSConnectionPool(host='mintransporte.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='mintransporte.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
minvu.cl,"HTTPSConnectionPool(host='minvu.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='minvu.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
minyu-net.com,"HTTPSConnectionPool(host='minyu-net.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='minyu-net.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
mise.gov.it,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
missbloom.bg,"HTTPSConnectionPool(host='missbloom.bg', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0f5760c790>, 'Connection to missbloom.bg timed out. (connect timeout=2)'))","HTTPConnectionPool(host='missbloom.bg', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0f5a826c40>, 'Connection to missbloom.bg timed out. (connect timeout=2)'))"
missmundo.nl,"HTTPSConnectionPool(host='missmundo.nl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f590745e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='missmundo.nl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f54468b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
misumi-ec.com,"HTTPSConnectionPool(host='misumi-ec.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f54b92eb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='misumi-ec.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f5941ab80>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mizuhobank.co.jp,"HTTPSConnectionPool(host='mizuhobank.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5a054b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mizuhobank.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f55e61730>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mj.pt,"HTTPSConnectionPool(host='mj.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5306dd60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mj.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f5ad49100>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mju.ac.th,"HTTPSConnectionPool(host='mju.ac.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f544a1880>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mju.ac.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f5813c550>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mku.edu.tr,"HTTPSConnectionPool(host='mku.edu.tr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5b600400>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mku.edu.tr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f521b91c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mladinska.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mlit.go.jp,"HTTPSConnectionPool(host='mlit.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f57c357f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mlit.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f53024490>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mlsi.gov.cy,"HTTPSConnectionPool(host='mlsi.gov.cy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5c678190>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mlsi.gov.cy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f55ef40a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mm.lv,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mmymagazine.com,"HTTPConnectionPool(host='www.mmymagazine.com', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.mmymagazine.com', port=80): Read timed out. (read timeout=2)"
mobilnisvet.com,"HTTPSConnectionPool(host='mobilnisvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='mobilnisvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mobmob.ro,"HTTPSConnectionPool(host='mobmob.ro', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mobmob.ro', port=443): Read timed out. (read timeout=2)"
moda.sk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
modacruz.com,"HTTPSConnectionPool(host='www.modacruz.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.modacruz.com', port=443): Read timed out. (read timeout=2)"
modenaora.it,"HTTPSConnectionPool(host='modenaora.it', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='modenaora.it', port=80): Read timed out. (read timeout=2)"
moe.edu.sg,"HTTPSConnectionPool(host='moe.edu.sg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f592308b0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='moe.edu.sg', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f5ac8b070>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
moe.go.th,"HTTPSConnectionPool(host='moe.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f521c1fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moe.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f54b63d90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moe.gov.jo,"HTTPSConnectionPool(host='moe.gov.jo', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='moe.gov.jo', port=443): Read timed out. (read timeout=2)"
moe.gov.lk,"HTTPSConnectionPool(host='moe.gov.lk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='moe.gov.lk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
moe.gov.my,"HTTPSConnectionPool(host='moe.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: TLSV1_UNRECOGNIZED_NAME] tlsv1 unrecognized name (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.moe.gov.my', port=443): Read timed out. (read timeout=2)"
mof.gov.vn,"HTTPSConnectionPool(host='mof.gov.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mof.gov.vn', port=443): Read timed out. (read timeout=2)"
mofa.go.jp,"HTTPSConnectionPool(host='mofa.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f5be157f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mofa.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f59bd7340>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moh.gov.bh,"HTTPSConnectionPool(host='moh.gov.bh', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f52e6bc40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moh.gov.bh', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0f5ab5a2b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moh.gov.my,"HTTPSConnectionPool(host='moh.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'moh.gov.my' doesn't match '*.moh.gov.my'"")))","HTTPSConnectionPool(host='moh.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'moh.gov.my' doesn't match '*.moh.gov.my'"")))"
moh.gov.vn,"HTTPSConnectionPool(host='moh.gov.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='moh.gov.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
mohre.gov.ae,"HTTPSConnectionPool(host='mohre.gov.ae', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mohre.gov.ae', port=443): Read timed out. (read timeout=2)"
moi.gov.ae,"HTTPSConnectionPool(host='moi.gov.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='moi.gov.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
